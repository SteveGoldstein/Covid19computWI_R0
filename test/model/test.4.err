
Attaching package: ‘dplyr’

The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──
✔ tibble  2.1.3     ✔ purrr   0.3.3
✔ tidyr   1.0.2     ✔ stringr 1.4.0
✔ readr   1.3.1     ✔ forcats 0.4.0
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
Default config will estimate R on weekly sliding windows.
    To change this change the t_start and t_end arguments. 
There were 50 or more warnings (use warnings() to see the first 50)
