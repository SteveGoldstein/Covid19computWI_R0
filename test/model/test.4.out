A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "1 The current R_0 in  Adams  is:  3.35 with 95% Credibility Interval ( 0.172 , 10.037 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "2 The current R_0 in  Ashland  is:  4.732 with 95% Credibility Interval ( 0.243 , 14.177 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "3 The current R_0 in  Barron  is:  5.033 with 95% Credibility Interval ( 0.894 , 11.937 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "4 The current R_0 in  Bayfield  is:  4.998 with 95% Credibility Interval ( 0.256 , 14.972 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "5 The current R_0 in  Brown  is:  1.263 with 95% Credibility Interval ( 1.185 , 1.344 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "6 The current R_0 in  Buffalo  is:  11.735 with 95% Credibility Interval ( 3.199 , 24.627 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "8 The current R_0 in  Calumet  is:  1.609 with 95% Credibility Interval ( 0.839 , 2.58 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "9 The current R_0 in  Chippewa  is:  3.815 with 95% Credibility Interval ( 1.79 , 6.454 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "10 The current R_0 in  Clark  is:  0.978 with 95% Credibility Interval ( 0.267 , 2.053 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "11 The current R_0 in  Columbia  is:  7.245 with 95% Credibility Interval ( 3.155 , 12.695 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "12 The current R_0 in  Crawford  is:  10.007 with 95% Credibility Interval ( 3.943 , 18.321 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "13 The current R_0 in  Dane  is:  0.581 with 95% Credibility Interval ( 0.394 , 0.799 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "14 The current R_0 in  Dodge  is:  0.905 with 95% Credibility Interval ( 0.491 , 1.422 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "15 The current R_0 in  Door  is:  2.579 with 95% Credibility Interval ( 1.211 , 4.363 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "16 The current R_0 in  Douglas  is:  2.47 with 95% Credibility Interval ( 0.439 , 5.859 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "17 The current R_0 in  Dunn  is:  8.684 with 95% Credibility Interval ( 3.422 , 15.898 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "18 The current R_0 in  Eau Claire  is:  3.601 with 95% Credibility Interval ( 2.358 , 5.058 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "19 The current R_0 in  Florence  is:  4.967 with 95% Credibility Interval ( 0.255 , 14.879 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "20 The current R_0 in  Fond du Lac  is:  3.038 with 95% Credibility Interval ( 1.873 , 4.433 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "22 The current R_0 in  Grant  is:  2.067 with 95% Credibility Interval ( 1.448 , 2.776 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "23 The current R_0 in  Green  is:  4.272 with 95% Credibility Interval ( 2.318 , 6.709 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "24 The current R_0 in  Green Lake  is:  3.046 with 95% Credibility Interval ( 1.041 , 5.905 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "25 The current R_0 in  Iowa  is:  6.755 with 95% Credibility Interval ( 2.308 , 13.095 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "26 The current R_0 in  Iron  is:  4.822 with 95% Credibility Interval ( 0.247 , 14.444 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "27 The current R_0 in  Jackson  is:  2.261 with 95% Credibility Interval ( 0.116 , 6.774 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "28 The current R_0 in  Jefferson  is:  0.341 with 95% Credibility Interval ( 0.093 , 0.715 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "29 The current R_0 in  Juneau  is:  1.336 with 95% Credibility Interval ( 0.582 , 2.342 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "30 The current R_0 in  Kenosha  is:  1.853 with 95% Credibility Interval ( 1.651 , 2.064 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "31 The current R_0 in  Kewaunee  is:  3.094 with 95% Credibility Interval ( 1.83 , 4.627 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "32 The current R_0 in  La Crosse  is:  2.66 with 95% Credibility Interval ( 1.159 , 4.661 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "33 The current R_0 in  Lafayette  is:  4.056 with 95% Credibility Interval ( 1.598 , 7.424 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "36 The current R_0 in  Manitowoc  is:  1.892 with 95% Credibility Interval ( 0.888 , 3.201 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "37 The current R_0 in  Marathon  is:  4.387 with 95% Credibility Interval ( 1.729 , 8.032 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "38 The current R_0 in  Marinette  is:  3.049 with 95% Credibility Interval ( 1.431 , 5.159 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "39 The current R_0 in  Marquette  is:  4.203 with 95% Credibility Interval ( 0.216 , 12.591 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "40 The current R_0 in  Menominee  is:  4.997 with 95% Credibility Interval ( 0.256 , 14.969 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "41 The current R_0 in  Milwaukee  is:  1.232 with 95% Credibility Interval ( 1.154 , 1.312 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "42 The current R_0 in  Monroe  is:  1.025 with 95% Credibility Interval ( 0.053 , 3.07 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "43 The current R_0 in  Oconto  is:  3.065 with 95% Credibility Interval ( 1.923 , 4.425 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "44 The current R_0 in  Oneida  is:  4.675 with 95% Credibility Interval ( 0.24 , 14.004 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "45 The current R_0 in  Outagamie  is:  1.92 with 95% Credibility Interval ( 1.382 , 2.53 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "46 The current R_0 in  Ozaukee  is:  1.527 with 95% Credibility Interval ( 0.828 , 2.397 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "48 The current R_0 in  Pierce  is:  1.104 with 95% Credibility Interval ( 0.196 , 2.618 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "49 The current R_0 in  Polk  is:  6.497 with 95% Credibility Interval ( 1.154 , 15.41 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "50 The current R_0 in  Portage  is:  3.063 with 95% Credibility Interval ( 0.835 , 6.428 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "51 The current R_0 in  Price  is:  4.822 with 95% Credibility Interval ( 0.247 , 14.445 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "52 The current R_0 in  Racine  is:  1.944 with 95% Credibility Interval ( 1.743 , 2.153 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "53 The current R_0 in  Richland  is:  1.192 with 95% Credibility Interval ( 0.325 , 2.503 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "54 The current R_0 in  Rock  is:  1.464 with 95% Credibility Interval ( 1.258 , 1.682 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "55 The current R_0 in  Rusk  is:  3.162 with 95% Credibility Interval ( 0.162 , 9.474 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "56 The current R_0 in  St. Croix  is:  4.031 with 95% Credibility Interval ( 2.261 , 6.216 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "57 The current R_0 in  Sauk  is:  2.1 with 95% Credibility Interval ( 1.482 , 2.806 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "58 The current R_0 in  Sawyer  is:  2.015 with 95% Credibility Interval ( 0.358 , 4.779 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "59 The current R_0 in  Shawano  is:  1.845 with 95% Credibility Interval ( 0.918 , 3.032 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "60 The current R_0 in  Sheboygan  is:  2.941 with 95% Credibility Interval ( 1.778 , 4.341 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "62 The current R_0 in  Trempealeau  is:  3.156 with 95% Credibility Interval ( 1.078 , 6.117 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "63 The current R_0 in  Vernon  is:  2.437 with 95% Credibility Interval ( 0.433 , 5.78 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "64 The current R_0 in  Vilas  is:  4.986 with 95% Credibility Interval ( 0.256 , 14.937 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "65 The current R_0 in  Walworth  is:  1.112 with 95% Credibility Interval ( 0.887 , 1.358 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "66 The current R_0 in  Washburn  is:  3.913 with 95% Credibility Interval ( 0.201 , 11.722 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "67 The current R_0 in  Washington  is:  1.216 with 95% Credibility Interval ( 0.682 , 1.875 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "68 The current R_0 in  Waukesha  is:  0.909 with 95% Credibility Interval ( 0.7 , 1.14 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "69 The current R_0 in  Waupaca  is:  1.035 with 95% Credibility Interval ( 0.282 , 2.172 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "70 The current R_0 in  Waushara  is:  14.53 with 95% Credibility Interval ( 3.96 , 30.492 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "71 The current R_0 in  Winnebago  is:  2.971 with 95% Credibility Interval ( 2.196 , 3.842 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "72 The current R_0 in  Wood  is:  5 with 95% Credibility Interval ( 0.256 , 14.978 )"
A discrete distribution
  name: gamma
  parameters:
    shape: 4.8659169550173
    scale: 1.54133333333333
[1] "73 The current R_0 in  Wisconsin  is:  1.398 with 95% Credibility Interval ( 1.351 , 1.445 )"
Warning messages:
1: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
2: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
3: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
4: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
5: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
6: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
7: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
8: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
9: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
10: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
11: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
12: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
13: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
14: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
15: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
16: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
17: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
18: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
19: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
20: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
21: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
22: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
23: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
24: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
25: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
26: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
27: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
28: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
29: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
30: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
31: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
32: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
33: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
34: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
35: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
36: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
37: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
38: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
39: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
40: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
41: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
42: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
43: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
44: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
45: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
46: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
47: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
48: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
49: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
50: In estimate_R_func(incid = incid, method = method, si_sample = si_sample,  ... :
  You're estimating R too early in the epidemic to get the desired
            posterior CV.
